frame/alliance/src/lib.rs

```diff
@@ -64,10 +64,11 @@
 //!   pass in order to retire.
 //! - `retire` - Retire from the Alliance and release the caller's deposit.
 //!
-//! #### For Members (Founders/Fellows)
+//! #### For Voting Members
 //!
 //! - `propose` - Propose a motion.
 //! - `vote` - Vote on a motion.
+//! - `veto` - Veto on a motion about `set_rule` and `elevate_ally`.
```

Comment:
I am a bit worried about allow any single member to veto. This was less an issue previously because founders is a fixed set of members.

We should either only allow FoundingFellow to veto, or simply remove this. Otherwise this effectively requires unanimously aye to pass any motion.